receiver/prometheusexecreceiver/factory_test.go

```diff
@@ -35,6 +35,11 @@ import (
 )
 
 func TestCreateTraceAndMetricsReceiver(t *testing.T) {
+	// Ensure default values are not 0 to prevent receiver regression error
+	timeZeroValue := model.Duration(0 * time.Second)
+	assert.NotEqual(t, timeZeroValue, defaultCollectionInterval)
+	assert.NotEqual(t, timeZeroValue, defaultTimeoutInterval)
```

Comment:
Those are constants, no need to test them. It would not be worth testing them.

Reply:
yes you are right, I removed the tests